react/get_env.sh

```diff
@@ -55,30 +55,31 @@ if [ -n "$1" ] && [ "$1" = "unset" ]; then
-cog_user_pool_id=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_COG_USER_POOL_ID' | jq -r .Parameter.Value)
+# Some parameter is taken from data-portal-client
+# Reference: https://github.com/umccr/data-portal-client
+cog_user_pool_id=$(aws ssm get-parameter --name '/data_portal/client/cog_user_pool_id' | jq -r .Parameter.Value)
 if [[ "$cog_user_pool_id" == "" ]]; then
   echo "Halt, No valid AWS login session found. Please 'aws sso login --profile dev && export AWS_PROFILE=dev'"
   return 1
 fi
-cog_identity_pool_id=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_COG_IDENTITY_POOL_ID' | jq -r .Parameter.Value)
-cog_app_client_id_local=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_COG_APP_CLIENT_ID_LOCAL' | jq -r .Parameter.Value)
-cog_app_client_id_stage=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_COG_APP_CLIENT_ID_STAGE' | jq -r .Parameter.Value)
+cog_app_client_id_local=$(aws ssm get-parameter --name '/data_portal/client/cog_app_client_id_local' | jq -r .Parameter.Value)
+cog_app_client_id_stage=$(aws ssm get-parameter --name '/sscheck/client/cog_app_client_id_stage' | jq -r .Parameter.Value)
 
-oauth_domain=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_OAUTH_DOMAIN' | jq -r .Parameter.Value)
-oauth_redirect_in_local=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_OAUTH_REDIRECT_IN_LOCAL' | jq -r .Parameter.Value)
-oauth_redirect_out_local=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_OAUTH_REDIRECT_OUT_LOCAL' | jq -r .Parameter.Value)
-oauth_redirect_in_stage=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_OAUTH_REDIRECT_IN_STAGE' | jq -r .Parameter.Value)
-oauth_redirect_out_stage=$(aws ssm get-parameter --name '/samplesheet-check/fe/react/REACT_APP_OAUTH_REDIRECT_OUT_STAGE' | jq -r .Parameter.Value)
+oauth_domain=$(aws ssm get-parameter --name '/data_portal/client/oauth_domain' | jq -r .Parameter.Value)
+oauth_redirect_in_local=$(aws ssm get-parameter --name '/data_portal/client/oauth_redirect_in_local' | jq -r .Parameter.Value)
+oauth_redirect_out_local=$(aws ssm get-parameter --name '/data_portal/client/oauth_redirect_out_local' | jq -r .Parameter.Value)
+oauth_redirect_in_stage=$(aws ssm get-parameter --name '/sscheck/client/oauth_redirect_in_stage' | jq -r .Parameter.Value)
+oauth_redirect_out_stage=$(aws ssm get-parameter --name '/sscheck/client/oauth_redirect_out_stage' | jq -r .Parameter.Value)
 
-api_url=$(aws ssm get-parameter --name '	/samplesheet-check/be/rest-api' | jq -r .Parameter.Value)
+lambda_api_domain=$(aws ssm get-parameter --name '/sscheck/lambda-api-domain' | jq -r .Parameter.Value)
+bucket_name=$(aws ssm get-parameter --name '/sscheck/bucket_name' | jq -r .Parameter.Value)
 
-export REACT_APP_API_URL=$api_url
+export REACT_APP_BUCKET_NAME=$bucket_name
+export REACT_APP_LAMBDA_API_DOMAIN=$lambda_api_domain
 export REACT_APP_STAGE=dev
```

Comment:
may be make is parameterise? it will become `REACT_APP_STAGE=prod` later, if we still push deployment from local...

Reply:
Added at `commit b760c56bf004a7e745d3a92e84ba27c36b98cd2e`